### PR TITLE
Add copy for the health section

### PR DESCRIPTION
**Pull request by ghost**
Adds copy for the health section. I also had to make some changes to the dependencies to get things running locally.

@alexgirard Thoughts on removing the long/concise version stuff? It seems like extra work that we don't have the time for. I can remove all of the code references if you agree.

### CODE REVIEW

File: content/health.html [hunk 1]
```diff
@@ -126,94 +102,86 @@ <h3>Imposter Syndrome</h3>
...
 <p class="half">
   <span class="big">
     Active bodies and active minds.
   </span>
-  Around 40% of respondents would find some time to exercise throughout the week,
-  while over half of respondents would exercise less frequently than that over the months and terms.
-  <br/><br/>
-  <!-- It's difficult to find a regular routine while moving around for school and coop. It's important to find a schedule that works for you, and that you can enjoy. -->
+  All respondents reported exercising at least once a week. 
   It's important to keep our bodies active as it can boost our overall health. In certain cases, aerobic exercise has been shown to reduce anxiety and depression.
```
Comment (RobbieZhuang) [inline content/health.html:129]: Wondering if this line is relevant?

File: content/health.html [hunk 2]
```diff
@@ -126,94 +102,86 @@ <h3>Imposter Syndrome</h3>
...
   <div id="exercise-frequency"></div>
 </div>
 
 <p class="half">
     <span class="big">
       Strengthening our lungs and muscles.
     </span>
-    45% of respondents reported doing strength-related exercises, like weight-training and calisthenics. 
-    And 38% reported doing aerobic exercises like walking, running, hiking, biking etc. Anything to get the heart pumping.
+    Respondents were involved in a wide variety of exercise types from running and weightlifting to rock climbing and muay thai.
```
Comment (RobbieZhuang) [inline content/health.html]: instead of exercise types, maybe "activities"?

**Review by RobbieZhuang — APPROVED** [review]
Looks good! Just a few suggestions to the copy. Also looks like there's a conflict in `build/bundle.min.js`

**Comment by RobbieZhuang** [comment]
I'm down to yeet the concise version, it doesn't look any shorter than the og one

**Comment by ghost** [comment]
@alexgirard Could you look at the changes to package.json and confirm you have no problems with them?

**Comment by RobbieZhuang** [comment]
Ooh I noticed that some of 2+ word activities don't have a space between them - think it was from a previous PR but if you could patch that in this one that'd be :pray:
image

Also this branch with the updated packages works for me lgtm

**Pull request merged**